erpnext/templates/pages/help.html

```diff
@@ -1,19 +1,60 @@
 {% extends "templates/web.html" %}
 
-{% block title %} {{ _("Help") }} {% endblock %}
-
-{% block header %}<h1>{{ _("Help") }}</h1>
+{% block content %}
+<section class="jumbotron" style="{{bg }}">
+	<div class='container' style="text-align: {{ align_greeting }};">
+		<h1 class="h1">{{ greeting_text }}</h1>
+		<p class="mt-4">
+			<div class="input-group shadow-sm" style="width:50%; margin-left: {{ align_search_box }}">
```

Comment:
I wasn't able to figure out a way to use the jinja variable inside the style tag to change the value for a particular class. If you have an idea for it, it would be helpful.

Reply:
You can use classes, inline styles are not to be used anywhere